modules/gles3/functional/es3fNegativeShaderApiTests.cpp

```diff
@@ -378,45 +380,47 @@ void NegativeShaderApiTests::init (void)
 
 			glDeleteShader(shader);
 		});
-	ES3F_ADD_API_CASE(get_program_binary, "Invalid glGetProgramBinary() usage",
-		{
-			glu::ShaderProgram				program			(m_context.getRenderContext(), glu::makeVtxFragSources(vertexShaderSource, fragmentShaderSource));
-			glu::ShaderProgram				programInvalid	(m_context.getRenderContext(), glu::makeVtxFragSources(vertexShaderSource, ""));
-			GLenum							binaryFormat	= -1;
-			GLsizei							binaryLength	= -1;
-			GLint							binaryPtr		= -1;
-			GLint							bufSize			= -1;
-			GLint							linkStatus		= -1;
-
-			m_log << TestLog::Section("", "GL_INVALID_OPERATION is generated if bufSize is less than the size of GL_PROGRAM_BINARY_LENGTH for program.");
-			glGetProgramiv		(program.getProgram(), GL_PROGRAM_BINARY_LENGTH,	&bufSize);
-			expectError			(GL_NO_ERROR);
-			glGetProgramiv		(program.getProgram(), GL_LINK_STATUS,				&linkStatus);
-			m_log << TestLog::Message << "// GL_PROGRAM_BINARY_LENGTH = " << bufSize << TestLog::EndMessage;
-			m_log << TestLog::Message << "// GL_LINK_STATUS = " << linkStatus << TestLog::EndMessage;
-			expectError			(GL_NO_ERROR);
-
-			glGetProgramBinary	(program.getProgram(), 0, &binaryLength, &binaryFormat, &binaryPtr);
-			expectError			(GL_INVALID_OPERATION);
-			if (bufSize > 0)
+		if(glu::isContextTypeES(m_context.getRenderContext().getType())){
+		ES3F_ADD_API_CASE(get_program_binary, "Invalid glGetProgramBinary() usage",
```

Comment:
Coding style: please indent this section properly and have a space between if and '('.

Reply:
Done